### ISSUE TITLE
Photo Collection - preserve order for albums

### ISSUE THREAD

**Issue by jcotton42** [MobiusSync/MobiusSync#223]
Currently the photo collection sync splits into folders and names the files for photos by their date. When syncing an album I would like the option to instead keep all the photos in one folder, naming them zero-padded (e.g. 0000 to 9999) in the same order as the album. I custom sort a lot of my albums, and having this feature would make Mobius Sync a better way to have a copy of those albums.

**Comment by SimonPickup** [comment]
I wasn't aware that the photo order within albums could be manually controlled, but I can see why you have made this request. I will have to investigate whether this order is available via the API.

However one problem with numerical seqeuncing, is that when you delete/insert/reorder photos, all the files would be renamed. (I understand Synthing may sometimes (but not always) be smart enough to avoid resending all the file contents.)

**Comment by jcotton42** [comment]
> However one problem with numerical seqeuncing, is that when you delete/insert/reorder photos, all the files would be renamed.

Yeah, the potential for a mass resend is unfortunate, but fortunately I don’t do too much shuffling of existing photos.

Thanks for looking into this by the way!

**Comment by ianroakes** [comment]
Adding another voice to this! I would also love to be able to sync all photo collections to one folder instead of splitting them up into individual nested folders by date. Thanks!
